reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+      parameters:
+      - $ref: '#/parameters/fields'
+      - $ref: '#/parameters/limit'
+      - $ref: '#/parameters/page'
+      - $ref: '#/parameters/since_id'
+      - $ref: '#/parameters/status'
+      - $ref: '#/parameters/ids'
+    
+      tags:
+        - Get
+
+      responses:
+        '200':
+          description: List of DraftOrder
+          schema:
+            $ref: '#/definitions/DraftOrder'
+        
+        '400':
+          description: General API error
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+       
+          
+            
+      x-code-samples:
+        - lang: curl
+          source:  |
+            curl -X GET -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders.json"
+      
+        - lang: Ruby
+          source: |
+              # load the gem
+              require 'swagger_client'
+              
+              api_instance = SwaggerClient::GetApi.new
+              
+              opts = { 
+                fields: "fields_example", # String | Comma-separated list of fields to include in the response
+                limit: 789, # Integer | Page size
+                page: 789 # Integer | Page to show
+              }
+              
+              begin
+                result = api_instance.admin_draft_orders_json_get(opts)
+                p result
+              rescue SwaggerClient::ApiError => e
+                puts "Exception when calling GetApi->admin_draft_orders_json_get: #{e}"
+              end
+            
+            
+  
+    post:
+      summary: Create a draft order
+      description: |
+          
+          Using the Draft Orders API you can create draft orders using product variant line items, or your own custom line items.  This enables you to create one-off products that exist outside of your regular inventory management system.  To create a product variant draft order, simply specify a `variant_id` `quantity` and `discount` (if applicable).  Create a custom line item by supplying `title`, `price`, `taxable` and `quantity`. 
+          
+        
+          | `Note`| The array of line items on a draft order must not contain a `variant_id` more than once. The data copied over from a variant upon creation of a line item is never subsequently updated.  |
+          |---|-------------|
+          
+          #### About Custom Shipping Lines
+          
+            Using the Draft Orders API provides you with the flexibility to fulfill order shipments either using Shopify or outside of Shopify.  To fulfill your order within Shopify, you can send the `handle` of the `shipping_rate` set via the Shopify admin.  For a custom shipping line, simply specify a `shipping_line` object with no `handle`.  This is covered in more detail in the endpoint calls below.  
```

Comment:
Shipping rates might not always be tied to fulfillments, since you might have a product fulfilled by vendor XYZ but the rate is provided by USPS. It doesn't 100% work that way in shopify right now since we can't have multi location but there's something about the way 'fulfill' is used in this sentence that seems incorrect.